src/Microsoft.NetCore.Analyzers/Core/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1212,4 +1212,13 @@
   <data name="DoNotUseCreateEncryptorWithNonDefaultIVDescription" xml:space="preserve">
     <value>Symmetric encryption should always use a non-repeatable initialization vector to prevent dictionary attacks.</value>
   </data>
+  <data name="DoNotDecryptWithoutHash" xml:space="preserve">
+    <value>Do not decrypt without hash</value>
+  </data>
+  <data name="DoNotDecryptWithoutHashDescription" xml:space="preserve">
+    <value>To validate the integrity of cipher text, it's requested to comput hash before decyption.</value>
+  </data>
+  <data name="DoNotDecryptWithoutHashMessage" xml:space="preserve">
+    <value>Byte array '{2}' in '{3}' passed to CryptoStream '{0}' in '{1}', never has the hash computed.</value>
```

Comment:
>Byte array '{2}' in '{3}' passed to CryptoStream '{0}' in '{1}', never has the hash computed. [](start = 11, length = 93)

The byte array '{2}' in '{3}' might be passed to CryptoStream '{0}' in '{1}' before computing a hash.